### ISSUE TITLE
EnderSkills packet problem

### ISSUE THREAD

**Issue by SSKirillSS** [dmulloy2/ProtocolLib#943]
**Describe the bug**
tl;dr version: Using EnderSkills skill, **that spawns entity**, cause print huge stacktrace to the logs

Described here: https://github.com/Mohist-Community/Mohist/issues/535
Log: https://pastebin.com/wstpviLa

**To Reproduce**
1. Use Mohist (Hybrid server (Forge+Spigot/paper/etc))
2. Install ProtocolLib, HolographicDisplays, HolographicExtension and PlaceholderAPI (2.10.6) to the plugins folder
3. Install EnderSkills (https://www.curseforge.com/minecraft/mc-mods/ender-skills) to the mods folder
4. Joint to the server, learn and cast skill
5. Check console

**Version Info**
https://pastebin.com/qE6HbCrV
